backend/corpora/common/corpora_orm.py

```diff
@@ -183,32 +183,32 @@ class DbProject(Base):
     data_submission_policy_version = Column(String, nullable=False)
 
     # Relationships
-    links = relationship("DbProjectLink", back_populates="project", cascade="all, delete-orphan")
-    datasets = relationship("DbDataset", back_populates="project", cascade="all, delete-orphan")
+    links = relationship("DbCollectionLink", back_populates="collection", cascade="all, delete-orphan")
+    datasets = relationship("DbDataset", back_populates="collection", cascade="all, delete-orphan")
 
 
-class DbProjectLink(Base):
+class DbCollectionLink(Base):
     """
-    Represents an external web link for DbProjects such as protocols and supplementary data repositories.
+    Represents an external web link for DbCollections such as protocols and supplementary data repositories.
     """
 
-    __tablename__ = "project_link"
+    __tablename__ = "collection_link"
```

Comment:
This will require a database migration

Reply:
this is the name of the table in the database. You will need to change the name of the table in the DB before doing doing this.